src/Pages/Myprofile/Myprofile.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import Tear from './Components/Tear';
+import Mytear from './Components/Mytear';
+import './Myprofile.scss';
+
+const NavCotent = [
```

Comment:
```suggestion
const NAV_CONTENT = [
```
- 오타 있습니다.
- 변수명은 navContent와 같이 카멜케이스로 쓰시되 변할 일 없는 정적인 변수는 대문자 스네이크 케이스로 작성해주세요.